app/src/app/not-found.tsx

```diff
@@ -0,0 +1,45 @@
+import OutboundArrowLink from "@/components/common/OutboundArrowLink"
+import Image from "next/image"
+import Link from "next/link"
+
+export default function NotFound() {
+  return (
+    <div className="flex flex-col items-center gap-6 mt-64">
+      <Image
+        src={"/assets/icons/sunny-stars-custom-gimp-edit.png"}
+        alt="Sunny Logo"
+        width={160}
+        height={160}
+      />
+      <h4 className="text-[#0F111A] font-semibold">
+        The page you're looking for doesn't exist.
```

Comment:
```suggestion
        The page you&apos;re looking for doesn&apos;t exist.
```

This is the reason the build fails